### DISCUSSION TITLE
how to get test siganal form cyton board

### DISCUSSION THREAD

**Discussion by ankur-gupta-29** [brainflow-dev/brainflow#678] [General]
how to get testsignal form cyton board .where i add test signal control commnads

-----------------------------------------------------------------------------------------------------------------------------------------------------
code--
-----------------------------------------------------------------------------------------------------------------
 board_shim.prepare_session()
        board_shim.config_board("x1045110X")
        board_shim.config_board("x2045110X")
        board_shim.config_board("x3045110X")
        board_shim.config_board("x4045110X")
        board_shim.config_board("x5045110X")
        board_shim.config_board("x6045110X")
        board_shim.config_board("x7045110X")
        board_shim.config_board("x8045110X")
        board_shim.start_stream(450000, args.streamer_params)

----------------------------------------------------------------------------------------------------------------------
openbci sdk details--

Test Signal Control Commands
0 - = p [ ]
Turn all available channels on, and connect them to internal test signal. These are useful for self test and calibration. For example, you can measure the internal noise by sending 0 which connects all inputs to an internal GND. If streaming, the stream will be stopped, the proper registers set on the ADS1299, and the stream will be resumed.

0 Connect to internal GND (VDD - VSS)

- Connect to test signal 1xAmplitude, slow pulse

= Connect to test signal 1xAmplitude, fast pulse

p Connect to DC signal

[ Connect to test signal 2xAmplitude, slow pulse

] Connect to test signal 2xAmplitude, fast pulse

Note: Not all possible internal test connections are implemented here

returns If not streaming, returns Success: Configured internal test signal.$$$, if streaming, there is no confirmation.

**Comment by Andrey1994** [comment]
Not sure I understand what the question is. You dont need to manually turn on the channels since there will be on by default, and call config_board with command from openbci docs between prepare_session and start_stream

**Reply by ankur-gupta-29** [reply]
i want to get test signal from cycton board so i send the command to cycton board board_shim.config_board("x1045110X") for setup the channel now i want to connect that channel to test signal of cyton board

**Reply by ankur-gupta-29** [reply]
so can i do that because board_shim.config_board("0") not working . it is showing some tab or space error

**Reply by Andrey1994** [reply]
idk, I only tested it with [ and ]
@retiutut

**Reply by ankur-gupta-29** [reply]
please can you share more details and code snippet of that?

**Reply by Andrey1994** [reply]
board.config_board("]") and nothing else, I dont understand why you are enabling each channel with some settings and what you are trying to test

**Reply by Andrey1994** [reply]
for openbci specific commands and testing protocol it can be better to ask at their forum since I am not an expert in it
